src/plugins/intel_cpu/src/nodes/if.cpp

```diff
@@ -21,27 +22,37 @@ If::PortMapHelper::PortMapHelper(const MemoryPtr &from, const std::deque<MemoryP
                                            const dnnl::engine& eng) : srcMemPtr(from), dstMemPtrs(to) {
     size = 0;
     if (srcMemPtr->getDesc().isDefined())
-        size = srcMemPtr->getSize();
+        size = srcMemPtr->getShape().getElementsCount();
+
+    // Backup dstMemPtrs
+    for (auto& ptr : dstMemPtrs) {
+        backupDstMemPtrs.push_back(ptr->getDescPtr()->clone());
```

Comment:
It's just not `backupDstMemPtrs` - not memory pointers. It's more about memory desc pointers. Probably, `originalDstMemDescs`is better

Reply:
Done, please help review again.